chapters/strings/count.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: recipe
+title: Count Substring
+chapter: Strings
+---
+## Problem
+
+You need to count occurrence of a search string within a message.
+
+## Solution
+
+{% highlight coffeescript %}
+count = (substr, string) ->
+  num = pos = 0
+  return 1/0 unless substr.length
```

Comment:
Input checking is not responsibility here.. Garbage in = garbage out.

needle, haystack will be ok?